routes/api-answer.js

```diff
@@ -0,0 +1,23 @@
+const db = require("../models");
+
+module.exports = function(app) {
+  app.post("/api/answer", function(req, res) {
+    db.Answer.create({
+      member_id: req.body.member_id,
+      question_id: req.body.question_id,
+      answer: req.body.answer
```

Comment:
Where is there a route for including team in here, so that there is a route for the game when a team is selected for play?